### ISSUE TITLE
Set tabwidth for verbatim block

### ISSUE THREAD

**Issue by pityhero233** [zim-desktop-wiki/zim-desktop-wiki#1767]
Hi community, the default tabwidth for verbatim text is 8. Is is possible to change it in the settings? Thanks :D

**Comment by jaap-karssenberg** [comment]
Verbatim is just a formatting style, so tab size will apply same as in the rest of the buffer. See "GUI style config file" at https://zim-wiki.org/manual/Help/Config_Files.html how to configure.

**Comment by pityhero233** [comment]
Sorry I tried this yesterday, but tabwidth in style.conf seems to be ignored ...
The style.conf is respected by Zim, as modification to color of verbatim text will be reflected; but regardless of how large I set indent, the tab is visually displayed as 8 characters.

Also should I set "indent size in pixels" anyways? since it's monospace, shouldn't I instead set tabwidth as number of characters it should display? Idk

Attached is the style.conf snippet. Thanks
```
[Tag code]
foreground=#1a192b
family=monospace
indent=4

[Tag pre]
foreground=#1a192b
family=monospace
wrap-mode=GTK_WRAP_NONE
indent=4
```
